### DISCUSSION TITLE
is there a way to disable ACK on received messages on a Socket CAN?

### DISCUSSION THREAD

**Discussion by frosst2906** [hardbyte/python-can#1726] [Q&A] [answered]
I am using a Lawicel CAN USB interface and I would like to know if with this library is it possible to enable/ disable ACK on RX messages. Thank you :)

**Comment by zariiii9003** [comment, marked as answer]
Not via python-can, but there's a `listen-only` parameter in SocketCAN:
`ip link set can0 type can listen-only on`

**Reply by frosst2906** [reply]
Thank you!
